Gopkg.toml

```diff
@@ -20,6 +20,8 @@
 #  name = "github.com/x/y"
 #  version = "2.4.0"
 
+# We don't want to end up with an old logrus import, only github.com/sirupsen/logrus should be used
+ignored = ["github.com/Sirupsen/logrus"]
```

Comment:
@hferentschik Isn't this change related to `Upgrade docker-machine to 0.13.0 ` ?

Reply:
No, containers/image and the upgrade of docker/docker and in fact everything else which depends on logrus. There are several.